libdnf/comps/group/query.cpp

```diff
@@ -0,0 +1,38 @@
+#include "libdnf/comps/group/query.hpp"
+#include "libdnf/comps/group/query_impl.hpp"
+#include "libdnf/comps/group/sack.hpp"
+
+extern "C" {
+#include <solv/pool.h>
+#include <libdnf/comps/comps.hpp>
+#include <libdnf/comps/comps_impl.hpp>
```

Comment:
This doesn't seem right to me.
Internal includes shouldn't be wrapped in `extern "C"`